lib/controller/python/controller/display.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ctypes
+from controller.wb import wb
+from controller.camera import Camera
+from controller.device import Device
+from typing import Union, List
+
+
+class Display(Device):
+    RGB = 3
+    RGBA = 4
+    ARGB = 5
+    BGRA = 6
+    ABGR = 7
+    wb.wb_display_image_copy.restype = ctypes.c_void_p
+    wb.wb_display_image_load.restype = ctypes.c_void_p
+    wb.wb_display_image_new.restype = ctypes.c_void_p
+    wb.wb_display_image_new.argtypes = [ctypes.c_void_p,
+                                        ctypes.c_int,
+                                        ctypes.c_int,
+                                        ctypes.c_void_p,
+                                        ctypes.c_int]
+
+    def __init__(self, name: Union[str, int]):
+        super().__init__(name)
+
+    def attachCamera(self, camera: Camera):
+        wb.wb_display_attach_camera(self._tag, camera._tag)
+
+    def detachCamera(self):
+        wb.wb_display_detach_camera(self._tag)
+
+    def drawLine(self, x1: int, y1: int, x2: int, y2: int):
+        wb.wb_display_draw_line(self._tag, int(x1), int(y1), int(x2), int(y2))
+
+    def drawOval(self, cx: int, cy: int, a: int, b: int):
+        wb.wb_display_draw_oval(self._tag, int(cx), int(cy), int(a), int(b))
+
+    def drawPixel(self, x: int, y: int):
+        wb.wb_display_draw_pixel(self._tag, int(x), int(y))
+
+    def drawPolygon(self, x: List[int], y: List[int]):
```

Comment:
Throw `TypeError: 'list' object cannot be interpreted as an integer` when doing:
```
p = [50, 60, 70, 650]
q = [10, 40, 50, 300]
self.camera_display.drawPolygon(p, q)
```
Which is working in develop